psyneulink/core/llvm/codegen.py

```diff
@@ -21,6 +21,453 @@
 from . import helpers
 from .debug import debug_env
 
+from enum import Enum
+class ArgumentType(Enum):
```

Comment:
this refers to types of other entities than arguments, no? maybe `VariableType` would be better?